example_project/example_project/management/commands/populate_redis.py

```
@@ -283,6 +285,43 @@ def create_test_data(self, redis_conn, db_num, key_count=100, options=None):
+                    "tags": [f"tag_{j}" for j in range(random.randint(1, 5))],
+                },
+                "nested": {
+                    "field1": f"value_{random.randint(1, 100)}",
+                    "field2": random.uniform(0.0, 100.0),
+                },
+            }
+            try:
+                redis_conn.json().set(json_key, "$", json_value)
+                json_keys_created += 1
+                created_keys += 1
+            except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)